sotodlib/coords/pmat.py

```diff
@@ -564,3 +619,19 @@ def _get_interpol_args(interpol):
         return {'interpol': interpol}
     assert interpol in [None, "nn", "nearest"], "Old so3g does not support interpolated mapmaking"
     return {}
+
+def check_pix_scheme(pix_scheme, geom=None):
```

Comment:
You might not need this function any more, or it might become private.  The docstring you have needs punctuation and refilling.  Or, remove it entirely; when a docstring for a short private function just restates in words what the code clearly manifests ... that's not super helpful and it can be left out.